build/_kpm-build.shade

```diff
@@ -0,0 +1,21 @@
+@{/*
+
+kpm-build
+    Builds a project.
+
+projectFile=''
+    Required. Path to the project.json to build.
+
+configuration=''
+    Optional. The configuration to build in. Defaults to 'Debug'.
+*/}
+
+default configuration = 'Debug'
+default build_options='${E("KPM_build_options")}'
```

Comment:
add `E("KPM_build_options"," --quiet");` to `--quiet` target in _k-standard-goals.shade